CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## v0.59.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.59.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.59.0) and the [opentelemetry-collector-contrib v0.59.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.59.0) releases.
+
```

Comment:
We should include some of the other enhancements, fixes, or breaking changes from https://github.com/signalfx/splunk-otel-collector/compare/v0.58.0...main.  Ideally, these changelog entries would be added by the authors.

Reply:
There's an enhancements section right below this that includes the new enhancements for v0.59.0, but since it was already committed since v0.58.0 it doesn't show up as added in this PR.

Is that what you were asking for, or is there more you're looking for?

Reply:
I was referring to the other changes from https://github.com/signalfx/splunk-otel-collector/compare/v0.58.0...main not yet in the changelog.  You may need to check each change to see if it will be relevant to include since the changelog will be automatically used for the release notes when the github release is created.